config/initializers/assets.rb

```diff
@@ -1,22 +1,12 @@
 # Be sure to restart your server when you modify this file.
 
-unless Rails.env.production?
-  Rails.application.config.assets.configure do |env|
-    env.cache = Sprockets::Cache::FileStore.new(
-      ENV.fetch("SPROCKETS_CACHE", "#{env.root}/tmp/cache/assets"),
-      Rails.application.config.assets.cache_limit,
-      env.logger
-    )
-  end
-end
-
 # Version of your assets, change this if you want to expire all your assets.
 Rails.application.config.assets.version = '1.0'
 
-# Add additional assets to the asset load path
+# Add additional assets to the asset load path.
 # Rails.application.config.assets.paths << Emoji.images_path
 
 # Precompile additional assets.
-# application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
-Rails.application.config.assets.precompile += %w( moderation.js )
-Rails.application.config.assets.precompile += %w( i18n.js )
```

Comment:
this is not required anymore because of webpacker, right?

Reply:
If I'm not mistaken we still use the asset pipeline for i18n. I'm also fairly certain rails admin still uses it

Reply:
Yes:
* `moderation.js` as an entrypoint doesn't exist anymore
* `i18n.js` is not used as an exported asset anymore either